homeassistant/components/zhong_hong/climate.py

```diff
@@ -65,6 +65,25 @@
     ZHONG_HONG_MODE_FAN_ONLY: HVACMode.FAN_ONLY,
 }
 
+# HA → zhong_hong
+FAN_MODE_MAP = {
+    "low": "LOW",
+    "medium": "MID",
+    "high": "HIGH",
+    "auto": "MIDHIGH",
+    "top": "HIGH",
+    "middle": "MID",
+    "focus": "MIDHIGH",
+    "diffuse": "MIDLOW",
+}
```

Comment:
Why does one dict have more items than the other?

You can also use `{v, k for k, v in FAN_MODE_MAP.items()}` for the reverse mapping

Also, let's use the constants like `FAN_LOW`